lookout/style/format/benchmarks/compare_quality_reports.py

```diff
@@ -0,0 +1,85 @@
+from typing import Optional, Union
+
+import pandas
+from tabulate import tabulate
+
+from lookout.style.format.benchmarks.top_repos_quality import FLOAT_PRECISION
+
+
+def _convert_cell_to_value(cell: str) -> Union[int, float, str]:
+    for type_ in (int, float):
+        try:
+            return type_(cell.strip())
+        except ValueError:
+            pass
+    cell = cell.strip()
+    return cell if cell else float("nan")
+
+
+def _quality_report_table_to_df(report_table_path: str) -> pandas.DataFrame:
+    table = []
+    separator_set = frozenset(":|-")
+    with open(report_table_path) as f:
+        for line in f:
+            if set(line.strip()) <= separator_set:
+                continue
+            cells = line.strip().split("|")
+            if not cells[0]:
```

Comment:
Do you mean `str.strip()`?

Reply:
no. the table assumes at least one column so at least two borders: `|`. So if you have the line:
```
| col1 | col2 |
```
and split it first and last, values are going to be empty.

Reply:
It is `strip`, not `split` 😉

Reply:
no, it should be like this. I do `line.strip()` no the previous step, so `cells[0].strip()` is not required.
Are you asking about this?

Reply:
I see, this is a list, not a string